composer.json

```diff
@@ -26,14 +26,14 @@
     ],
     "homepage": "https://www.doctrine-project.org/projects/lexer.html",
     "require": {
-        "php": "^7.1 || ^8.0"
+        "php": "^8.1"
     },
     "require-dev": {
-        "doctrine/coding-standard": "^9 || ^10",
-        "phpstan/phpstan": "^1.3",
-        "phpunit/phpunit": "^7.5 || ^8.5 || ^9.5",
+        "doctrine/coding-standard": "^10",
+        "phpstan/phpstan": "^1.9",
+        "phpunit/phpunit": "^9.5",
         "psalm/plugin-phpunit": "^0.18.3",
-        "vimeo/psalm": "^4.11 || ^5.0"
+        "vimeo/psalm": "^4.11"
```

Comment:
Why the downgrade?

Reply:
Bad rebase I think, let me fix it.

Reply:
Or maybe a brainfart when removing the pipes